pages/common/assets/sass/style.scss

```diff
@@ -0,0 +1,92 @@
+@import "../../../../ui/assets/default.scss";
```

Comment:
This feels odd. We should have common sass in one place, not arbitrarily in two places.